lib/js-file.js

```diff
@@ -245,26 +184,55 @@ JsFile.prototype = {
     getLastNodeToken: function(node) {
         return this.getTokenByRangeEnd(node.range[1]);
     },
+
     /**
      * Returns the first token before the given.
      *
      * @param {Object} token
+     * @param {Boolean} [includeComments=false]
```

Comment:
Yeah, @ariya would frown on our use of the boolean trap. However, this is more for pragmatism than anything else at the moment.